File: Level 0/코딩 기초 트레이닝/정사각형으로 만들기/solution.py
def solution(arr):
    rows, cols = (
        len(arr),
        len(arr[0]),
    )
    diff = abs(rows - cols)

    if rows > cols:
        for index in range(rows):
            for _ in range(diff):
                arr[index].append(0)
    elif rows < cols:
        for _ in range(diff):
            arr.append([0] * cols)

    return arr

File: Level 0/코딩 기초 트레이닝/정사각형으로 만들기/solution.js
function solution(arr) {
  const rows = arr.length;
  const cols = arr[0].length;
  const diff = Math.abs(rows - cols);

  if (rows > cols) {
    arr.forEach((_, i) => {
      for (let j = 0; j < diff; j++) {
        arr[i].push(0);
      }
    });
  } else if (rows < cols) {
    for (let i = 0; i < diff; i++) {
      arr.push(Array(cols).fill(0));
    }
  }

  return arr;
}
